Taxi/Taxi/Presenter/OnboardFirstPage.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  OnboardFirstPage.swift
+//  Taxi
+//
+//  Created by joy on 2022/06/15.
+//
+
+import SwiftUI
+
+struct OnboardFirstPage: View {
```

Comment:
```suggestion
struct OnboardFirstPageView: View {
```

Model인지 View인지 구분하기 쉬운 이름이 좋아보여요 ~

Reply:
네이밍 피드백 감사합니다 :)